results/brms_spatphylo.R

```diff
@@ -7,9 +7,9 @@ source('varcov.spatial_function.R')
 
 addTaskCallback(function(...) {set.seed(123);TRUE})
 
-glottolog_langs <- read.csv("Glottolog_Languages.csv.gz")
-datfra <- read.table(file = "BT_data.txt")
-tree <- read.nexus("pruned_tree.tree")
+glottolog_langs <- read.csv("Glottolog_Languages.csv")
+datfra <- read.table(file = "0001_or_2aKA/BT_data.txt")
```

Comment:
This has just broken the script for all other features. please revert all these changes and do what you want to do here elsewhere.